src/main/recorder.ts

```diff
@@ -596,6 +609,32 @@ type RecorderOptionsType = {
         const closestResolution = getClosestResolution(availableResolutions, res);
         this.setSettingOBS('Video', paramString, closestResolution);
     }
+
+    async assertNextSignal (value: string) {
```

Comment:
Don't forget to mark them `private` when they aren't supposed to be called externally.